nautilus_trader/data/aggregation.pxd

```diff
@@ -63,6 +63,7 @@ cdef class BarAggregator:
     cdef Logger _log
     cdef BarBuilder _builder
     cdef object _handler
+    cdef object _backup_handler
```

Comment:
Whats the purpose of the backup handler?

Reply:
It's to temporarily save the original handler while another one is used by the batch aggregation after a data response from request_bars. This avoid moving the handler to another domain like the data engine.